multiapps-controller-process/src/main/resources/org/cloudfoundry/multiapps/controller/process/deploy-app.bpmn

```diff
@@ -130,6 +126,25 @@
       </extensionElements>
       <conditionExpression xsi:type="tFormalExpression"><![CDATA[${appStateActionsToExecute.contains('START')}]]></conditionExpression>
     </sequenceFlow>
+    <callActivity id="bindUnbindServicesCallActivity" name="Bind Unbind Services Call Activity" flowable:async="true" calledElement="bindUnbindServiceSubProcess" flowable:calledElementType="key" flowable:inheritVariables="true" flowable:completeAsync="true" flowable:fallbackToDefaultTenant="false">
```

Comment:
You can generilize the naming to "Manage Application Service Bindings Call Activity" and propagate the wording.